app/forms/candidate_interface/becoming_a_teacher_form.rb

```diff
@@ -44,10 +39,8 @@ def save(application_form)
     end
 
     def presence_of_statement
```

Comment:
Do we still need this method?

Reply:
Good shout raising this – I glossed over it but it turns out we're not validating the presence of the statement at all. I just tested locally on main and you can submit an app without a personal statement 🤔